src/main/java/ui/issuepanel/FilterPanel.java

```diff
@@ -68,7 +83,55 @@ public FilterPanel(UI ui, IModel model, PanelControl parentPanelControl, int pan
                 getStyleClass().remove("panel-focused");
             }
         });
-        
+        setupKeyboardShortcuts();
+    }
+
+    private void setupKeyboardShortcuts() {
+        addEventHandler(KeyEvent.KEY_PRESSED, event -> {
+            if (MAXIMIZE_WINDOW.match(event)) {
+                ui.maximizeWindow();
+            }
+            if (MINIMIZE_WINDOW.match(event)) {
+                ui.minimizeWindow();
+            }
+            if (DEFAULT_SIZE_WINDOW.match(event)) {
+                ui.setDefaultWidth();
+            }
+            if (SWITCH_DEFAULT_REPO.match(event)) {
+                ui.switchDefaultRepo();
+            }
+            if (SWITCH_BOARD.match(event)) {
+                ui.getMenuControl().switchBoard();
+            }
+            if (JUMP_TO_FILTER_BOX.match(event)) {
+                setFocusToFilterBox();
+            }
+            if (JUMP_TO_FIRST_ISSUE.match(event)) {
+                listView.selectNthItem(1);
+            }
+            for (Map.Entry<Integer, KeyCodeCombination> entry : JUMP_TO_NTH_ISSUE_KEYS.entrySet()) {
+                if (entry.getValue().match(event)){
+                    event.consume();
+                    listView.selectNthItem(entry.getKey());
+                    break;
+                }
+            }
+        });
+    }
+    
+    private void setFocusToFilterBox() {
+        if (TestController.isTestMode()) {
+            ui.triggerEvent(new UIComponentFocusEvent(UIComponentFocusEvent.EventType.FILTER_BOX));
+        }
+        filterTextField.requestFocus();
+        filterTextField.setText(filterTextField.getText().trim());
+        filterTextField.positionCaret(filterTextField.getLength());
+
+        addEventHandler(KeyEvent.KEY_PRESSED, event -> {
```

Comment:
This looks like a bug (from a while back it seems). Every time we jump to the filter box this handler will be bound.